x/evm/keeper/send.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+)
+
+func (k *Keeper) EVMToEVMSend(ctx sdk.Context, fromAddress common.Address, toAddress common.Address, amount uint64) error {
+	if err := k.DebitAddress(ctx, fromAddress, amount); err != nil {
+		return err
+	}
+	return k.CreditAddress(ctx, toAddress, amount)
+}
+
+func (k *Keeper) BankToEVMSend(ctx sdk.Context, bankAddress sdk.AccAddress, toAddress common.Address, amount uint64) error {
+	// first transfer tokens to evm module account
```

Comment:
should we check some edge cases (like amount >= 0). Do we also allow arbitrary types (ikbc) or is everythign guaranteed to be in sei

Reply:
amount is uint so it will always be non-negative. Added checks for amount==0 to early return. Everything will be in usei w.r.t EVM